dev-gap/alnuth/alnuth-3.2.1.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gap-pkg
+
+DESCRIPTION="Algebraic number theory and an interface to PARI/GP"
```

Comment:
This package won't do much good if PARI/gp is not installed. Should it be a dependency (I assume you have a package me for it, surely Sage needs one?)

Reply:
Indeed, I added it. Thanks!